Apps/WebClient/src/ClientApp/src/views/ContactUsView.vue

```diff
@@ -1,93 +0,0 @@
-<script lang="ts">
-import Vue from "vue";
-import { Component } from "vue-property-decorator";
-
-import BreadcrumbComponent from "@/components/navmenu/BreadcrumbComponent.vue";
-import HealthlinkSidebarComponent from "@/components/timeline/HealthlinkSidebarComponent.vue";
```

Comment:
HealthlinkSidebarComponent.vue can also be deleted now, along with the 3 .png assets it references.

Reply:
Removed.